packages/nx/src/generators/tree.ts

```diff
@@ -7,6 +7,7 @@ import {
   removeSync,
   chmodSync,
 } from 'fs-extra';
+import { visitNotIgnoredFiles } from '../../../devkit';
```

Comment:
This import bring a circular dependency. How should I proceed if I want to use `visitNotIgnoredFiles` in this file ?